apps/ledger-live-desktop/src/renderer/screens/manager/DeviceStorage/CustomImageManagerButton.tsx

```diff
@@ -12,12 +13,11 @@ const CustomImageManagerButton: React.FC<Record<string, never>> = () => {
   }, []);
   return (
     <Flex flexDirection="row" columnGap={3} alignItems="center">
-      <Icons.ToolsMedium color="neutral.c80" size={16} />
```

Comment:
are you sure about this icon? I feel like it was used as a placeholder in Figma...
anyway we will do a pixel polish soon so this doesn't matter

Reply:
I'm not sure about anything, I just followed the figma